cmd/kured/main.go

```
@@ -245,7 +245,9 @@ type KubernetesBlockingChecker struct {
 }
 
 func (pb PrometheusBlockingChecker) isBlocked() bool {
-	alertNames, err := alerts.PrometheusActiveAlerts(pb.promURL, pb.filter)
+	prom, _ := alerts.PromClient{}.New(papi.Config{Address: prometheusURL})
```

Should we ignore the error here?